### PR TITLE
fix: fix editor output for set and maps

### PR DESCRIPTION
**Pull request by Ismailtlem**
- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Related to #44306

### CODE REVIEW

File: client/src/utils/format.js [hunk 1]
```diff
@@ -3,5 +3,13 @@ import { inspect } from 'util';
 export function format(x) {
   // we're trying to mimic console.log, so we avoid wrapping strings in quotes:
   if (typeof x === 'string') return x;
+  else if (x instanceof Set) {
+    return `Set(${x.size}) {${Array.from(x).join(', ')}}`;
+  } else if (x instanceof Map) {
+    return `Map(${x.size}) {${Array.from(
+      x.entries(),
+      ([k, v]) => `${k}=>${v}`
```
Comment (ShaunSHamilton) [inline client/src/utils/format.js:11]:
```suggestion
      ([k, v]) => `${k} => ${v}`
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Small change. It is a bit of a hacky approach, but it is _kilometres_ better than `{}`

**Comment by ojeytonwilliams** [comment]
Thanks @Ismailtlem.  I think we need to fix `format` too, because it's using node's `util` in the tests, but `util/` in production :(  Also we should explicitly install the 'util' lib in the client and fallback to it. i.e update the following:
https://github.com/freeCodeCamp/freeCodeCamp/blob/d58e5e5bddcd14be89ac2e2d240205edbd05e536/client/gatsby-node.js#L232

Let me know if you're interested in tackling that.  If not, no worries, I'll get to it one of these days!

**Comment by nhcarrigan** [comment]
@ojeytonwilliams to clarify, is that a blocker on this PR?

**Comment by ojeytonwilliams** [comment]
No, sorry for being vague. This PR shouldn't be blocked for that. Tests can come separately.

**Review by naomi-lgbt — APPROVED** [review]
image

Looks good to me!

**Review by Sembauke — APPROVED** [review]

**Review by ShaunSHamilton — APPROVED** [review]
Thank you for your contribution to the page! 👍
We are happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**